src/main/java/org/apache/flink/benchmark/InputBenchmark.java

```diff
@@ -34,31 +35,33 @@
 
 @OperationsPerInvocation(value = InputBenchmark.RECORDS_PER_INVOCATION)
 public class InputBenchmark extends BenchmarkBase {
-
 	public static final int RECORDS_PER_INVOCATION = 15_000_000;
 	private static final long CHECKPOINT_INTERVAL_MS = 100;
 
 	public static void main(String[] args)
-		throws RunnerException {
+			throws RunnerException {
 		Options options = new OptionsBuilder()
-			.verbosity(VerboseMode.NORMAL)
-			.include(".*" + InputBenchmark.class.getCanonicalName() + ".*")
-			.build();
+				.verbosity(VerboseMode.NORMAL)
+				.include(".*" + InputBenchmark.class.getCanonicalName() + ".*")
+				.build();
 
 		new Runner(options).run();
 	}
 
+	@Param({"LEGACY_SOURCE", "FLIP_27_SOURCE_BOUNDED", "FLIP_27_SOURCE_UNBOUNDED"})
```

Comment:
This will become a part of the benchmark name, so I would shorten it as much as possible.

```
{"LEGACY", "BOUNDED", "UNBOUNDED"}
```
```
{"LEGACY", "F27_BOUNDED", "F27_UNBOUNDED"}
```
?